index.bs

```diff
@@ -1681,9 +1682,14 @@ these steps:
 
 <div class="note domintro">
   : |handle| = await window . {{Window/showDirectoryPicker()}}
+  : |handle| = await window . {{Window/showDirectoryPicker()}}({ {{DirectoryPickerOptions/mode}}: 'read' })
   :: Shows a directory picker that lets the user select a single directory, returning a handle for
     the selected directory.
 
+  : handles = await window . {{Window/showDirectoryPicker()}}({ {{DirectoryPickerOptions/mode}}: 'readwrite' })
+  :: Shows a directory picker that lets the user select a single directory, returning a handle for
+    the selected directory. The user agent can combine read and write permission requests on this handle into one prompt.
```

Comment:
nit: perhaps mentioning something about that this will only return the directory handle if the user granted write permission might be helpful

Reply:
Should we mention the same thing in the `read` case above?

`Shows a directory picker that lets the user select a single directory, returning a handle for the selected directory if the user grants <mode> permission`

Reply:
I guess it wouldn't hurt. There is some overlap between this language and the language in the first paragraphs of https://wicg.github.io/file-system-access/#local-file-system-permissions, but these are more clearly meant for web developers, so repeating that info doesn't seem bad.